src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -145,7 +145,7 @@ public SortedSet(IEnumerable<T> collection, IComparer<T> comparer)
                     int index = 1;
                     for (int i = 1; i < count; i++)
                     {
-                        if (comparer.Compare(els[i], els[i - 1]) != 0)
+                        if (_comparer.Compare(els[i], els[i - 1]) != 0)
```

Comment:
Sure. Didn't think the overhead introduced by the extra layer of indirection to access the field would be tangible, but considering the comment overhead is 0 I'll switch to that :)